packages/appstore/src/components/cfds-listing/index.tsx

```diff
@@ -91,6 +100,25 @@ const CFDsListing = observer(() => {
     const { has_svg_accounts_to_migrate } = useMT5SVGEligibleToMigrate();
     const getAuthStatus = (status_list: boolean[]) => status_list.some(status => status);
 
+    const { getPlatformStatus } = useTradingPlatformStatus();
```

Comment:
Call the hook at the top level of the component. Our hooks first and then destructuring assignments, ... .